src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from "react";
+import { useCss } from "react-use";
+import classNames from "classnames";
+
+import { DEFAULT_VENUE_LOGO } from "settings";
+
+import { useRelatedVenues } from "hooks/useRelatedVenues";
+
+import "./ScheduleVenueDescription.scss";
+
+export interface ScheduleVenueDescriptionProps {
+  venueId: string;
+}
+
+export const ScheduleVenueDescription: FC<ScheduleVenueDescriptionProps> = ({
+  venueId,
+}) => {
+  const { sovereignVenue } = useRelatedVenues({
+    currentVenueId: venueId,
+  });
+
+  const venuePictureCssVars = useCss({
+    "--venue-picture--background-image": `url(${
+      sovereignVenue?.host?.icon ?? DEFAULT_VENUE_LOGO
+    })`,
+  });
```

Comment:
This might be cleaner if the value was extracted as a `const`

Reply:
extracted to `venueIcon` const: https://github.com/sparkletown/sparkle/pull/1426/commits/880f966fdbeb06065f5946456902e13410b88582